GovUk.Frontend.ExampleApp/Views/SearchResults/Index.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using Microsoft.Extensions.Configuration
+@inject IConfiguration config;
+@model TextInputViewModel
+@section scripts {
+
+    <script src="_Content/ThePensionsRegulator.Frontend/tpr/tpr-search.js" type="module"></script>
+}
+@{
+    ViewData["Title"] = "Q&As search results";
+}
+
+<span class="govuk-caption-xl">Example application</span>
+<div class="govuk-grid-row govuk-grid-row--tpr-divider">
+    <div class="govuk-grid-column govuk-grid-column-full">
+        <h1 class="govuk-heading-l">@ViewData["Title"]</h1>
+    </div>
+</div>
+
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column govuk-grid-column-two-thirds-from-desktop">
+        <p class="govuk-body">
+            The Q&As search results block can be added to any page where you want the user to be able to search the Q&A content accessible on <a class="govuk-link" href="https://www.thepensionsregulator.gov.uk/faq-landing-page">The Pensions Regulator website</a>. Currently this is focused on Q&As for Automatic Enrolement.
+        </p>
+
+        <p class="govuk-body">
+            The title and footer links can be overridden.
+        </p>
+
+        <p class="govuk-body">
+            This component relies on JavaScript to display content. If JavaScript is not available then this component will not display. The JavaScript file 'tpr-search.js' is provided as part of the 'ThePensionsRegulator.Frontend' nuget package. This will need to be included in any view that uses this component.
+        </p>
+
+        <p class="govuk-body">
+            The URL for the required API can be configured in appsettings.json file under the key "FaqApiUrl". This should point to the API that provides the Q&A content. The API should return a JSON response that matches the expected structure for the search results.
```

Comment:
Looks like the URL can be configured any way you like (which is good), and this example app uses the setting you mention. The key info is it's set on the tag helper.